recipes/dcase2022_task4_baseline/local/sed_trainer.py

```diff
@@ -751,3 +763,38 @@ def test_dataloader(self):
             drop_last=False,
         )
         return self.test_loader
+
+    def on_train_end(self) -> None:
+        # dump consumption
+        self.tracker_train.stop()
+        training_kwh = self.tracker_train._total_energy.kwh
+        with open(os.path.join(self.exp_dir, "training_codecarbon", "training_tot_kwh.txt"), "w") as f:
```

Comment:
Don't we need to do a 'print' or display something saying we saved the kWh at this location ?

Reply:
Agree, I would display the results as we do for the metrics.

Reply:
I can add a print in addition to this

Reply:
added